katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -0,0 +1,505 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_katello
+%global confdir hammer
+
+Summary: Katello command plugin for the Hammer CLI
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.11.0
+Release: 1%{?dist}
+Group:   Development/Languages
+License: GPLv3
+URL:     https://github.com/theforeman/hammer-cli-katello
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 0.0.17-3
+Obsoletes: rubygem-hammer_cli_gutterball
+%endif
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.5.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 1.0.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman_tasks) >= 0.0.3
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman_bootdisk)
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman_docker)
+
+BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+
+%description
+Katello command plugin for the Hammer CLI.
+
+%package doc
+Summary:   Documentation for %{pkg_name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}-doc}
+
+%description doc
+Documentation for %{pkg_name}
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - <<EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d
+install -m 755 .%{gem_instdir}/config/katello.yml %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d/katello.yml
```

Comment:
Also executable

Reply:
From what I can tell, this is true in all hammer packages. I'd prefer not to one off change a few and make this change across the board.